Library/Homebrew/utils/livecheck.rb

```diff
@@ -0,0 +1,28 @@
+# frozen_string_literal: true
+
+require "open3"
+
+module Livecheck
+  def livecheck_formula_response(formula_name)
+    ohai "- livecheck formula : #{formula_name}"
```

Comment:
```suggestion
    ohai "checking livecheck formula : #{formula_name}" 
```

or something? Describe what you're doing